charts/hdfs-namenode-k8s/values.yaml

```diff
@@ -39,9 +39,6 @@ zookeeperQuorum: zk-0.zk-svc.default.svc.cluster.local:2181,zk-1.zk-svc.default.
 # a standby namenode. Separated by the semicolon character.
 journalQuorum: hdfs-journalnode-0.hdfs-journalnode.default.svc.cluster.local:8485;hdfs-journalnode-1.hdfs-journalnode.default.svc.cluster.local:8485;hdfs-journalnode-2.hdfs-journalnode.default.svc.cluster.local:8485
 
-# Whether or not to enable pinning of namenode pods to labeled k8s cluster nodes.
```

Comment:
I found this flag is referred to by README.md. See https://github.com/vvbogdanov87/kubernetes-HDFS/blame/96720ecedf2d29d669b732059de3628c6782a9a2/charts/hdfs-namenode-k8s/README.md#L134

Can you please remove the section?